### DISCUSSION TITLE
自定义词云形状只能是方形吗

### DISCUSSION THREAD

**Discussion by wei-z-git** [he0119/nonebot-plugin-wordcloud#124] [Q&A]
自定义词云形状只能是方形吗，有没有可能设置心形等不规则形状

**Comment by he0119** [comment]
生成的图片应该只能是矩形，不过词云自己的形状是可以调整的。
比如 <https://amueller.github.io/word_cloud/auto_examples/masked.html#sphx-glr-auto-examples-masked-py>。
